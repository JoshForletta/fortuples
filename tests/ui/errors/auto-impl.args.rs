#[fortuples::auto_impl(not_expected)]
trait Test {}

fn main() {}
