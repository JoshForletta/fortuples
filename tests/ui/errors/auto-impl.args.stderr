error: `auto_impl` doesn't take arguments
 --> tests/ui/errors/auto-impl.args.rs:1:24
  |
1 | #[fortuples::auto_impl(not_expected)]
  |                        ^^^^^^^^^^^^
